app/src/main/java/de/danoeh/antennapod/fragment/swipeactions/TogglePlaybackStateSwipeAction.java

```diff
@@ -39,6 +39,10 @@ public void performAction(FeedItem item, Fragment fragment, FeedItemFilter filte
 
     @Override
     public boolean willRemove(FeedItemFilter filter, FeedItem item) {
-        return filter.showUnplayed || filter.showPlayed || filter.showNew;
+        if (item.getPlayState() == item.NEW) {
```

Comment:
```suggestion
        if (item.getPlayState() == FeedItem.NEW) {
```

Reply:
Oh, of course. :laughing:

I have just pushed another commit which makes this change.